node/full.go

```diff
@@ -136,9 +139,9 @@ func (qm *QitmeerFull) RegisterNotifyMgr() error {
 	return nil
 }
 
-func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config) error {
+func (qm *QitmeerFull) RegisterAccountService(cfg *config.Config, autoCollectOp chan types.AutoCollectUtxo) error {
 	// account manager
```

Comment:
the additional feature should handle by the configuration, instead of change the API.

Reply:
is not required to change the API definition here. 
acct.New(with cfg) is good enough , should not have to passing a runtime object.  if you really want the parameter, please reconsider the design. how the object life cycle is defined and why they depends on the each others.